config/routes.rb

```diff
@@ -1,5 +1,5 @@
 MANIACChess::Application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers => { :omniauth_callbacks => "users/omniauth_callbacks" }
```

Comment:
I was little hesitant to add the users controller since I wasn't sure how we wanted to handle it but I can go ahead and generate a users controller so routing doesn't fail for now.